client/src/frontend/components/journal/Metadata/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ContentBlock from "frontend/components/content-block/Block";
+
+function Metadata({ journal }) {
+  return (
+    <ContentBlock
+      block={{
+        attributes: {
+          renderable: true,
+          type: "Content::MetadataBlock",
+          title: "Journal info",
+          icon: "journals64"
+        }
+      }}
+      entity={journal}
+    />
+  );
+}
```

Comment:
I think the white space between this and volume groups is taller in design?

<img width="1408" alt="Screen Shot 2022-01-18 at 11 16 17 AM" src="https://user-images.githubusercontent.com/32903719/150004274-0d4de5bb-1f46-4413-9bfa-06c0b79b3a1e.png">

Reply:
Same below the metadata block. It looks to me like there's more white space in design.

Reply:
Yeah true. In these cases I'm just using the block padding defined on `.container`. Even though it departs from design a bit, I'm inclined to use global values like this rather than to start defining specific spacing for a particular detail view. Maybe where we should be heading is creating something like a `EntityDetail` component that lays out page elements with consistent spacing. Abstract it at that level, rather than to try to define it consistently on each route.